.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -150,6 +152,29 @@ private object Notifications : WPComPluginBuild(
 	""".trimIndent(),
 )
 
+private object CalypsoStats : WPComPluginBuild(
+	buildId = "WPComPlugins_CalypsoStats",
+	buildName = "CalypsoStats",
+	pluginSlug = "calypso-stats",
+	archiveDir = "./dist/",
+	docsLink = "PCYsg-elI-p2",
+	// This param is executed in bash right before the build script compares
+	// the build with the previous release version. The purpose of this code
+	// is to remove sources of randomness so that the diff operation only
+	// compares legitimate changes.
+	normalizeFiles = """
+		# Replace the old cache buster with the new one in the previous release html files.
+		if [ -f './release-archive/build_meta.json' ] ; then
+			old_cache_buster=`jq -r '.cache_buster' ./release-archive/build_meta.json`
+		else
+			old_cache_buster=`cat ./release-archive/cache-buster.txt`
+		fi
+
+		new_cache_buster=`jq -r '.cache_buster' ./dist/build_meta.json`
+		sed -i "s~${'$'}old_cache_buster~${'$'}new_cache_buster~g" release-archive/index.html release-archive/rtl.html
```

Comment:
This script might fail, since the HTML file doesn't exist. It might be worth having an HTML file eventually, especially if this project needs to be loaded standalone in an iframe on atomic for security reasons.

But yeah, you might need to remove this line to get it passing on trunk.

Reply:
Yep, you should be able to remove it.

Got it. Will you ultimately link to the assets in widgets.wp.com in Jetpack then?

Reply:
> Got it. Will you ultimately link to the assets in widgets.wp.com in Jetpack then?

That's correct.

Reply:
One thing the HTML wrapper helps with for notifications is linking all of the random webpack bundle filenames. That's definitely a solvable problem without the HTML wrapper, but just a thought. It ultimately depends how you need to consume it 👍